### PR TITLE
Repurposed nightmare insert + Lambda Changes

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
# About the pull request

Makes Lambda more nice and adds some more items in the lackluster areas, makes the Director's office more like a Director's office and not a rundown 3 table 1 paper bin office, with a fluff diary note in his Panic Room/Bedroom. 

Also uses an old nightmare/sprinkle of Big Red and repurposes it to a like Chance's, a moneybags vault which was mined out and created around the same time the new mining began southwest in the caves of the map

# Explain why it's good for the game

I made the Director's Office, Admin Office and Relax Room in Lambda more nice and added a fluff note in the Director's Safe Room

I repurposed an old nightmare insert, there is a roadmap saying to add more nightmare inserts so why not.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>
The Director's brand new office. An expanded Administration Office. More funded Relaxation Room. (Union mandated)

image

The Weyland-Yutani shared vault, with a dead PMC outside guarding it.

image

</details>

# Changelog
:cl:
add: Added a flavor diary entry from the Director of Lambda
mapadd: added back the old vault nightmare insert on Solaris which was removed due to the creation of static comms where it spawned
maptweak: tweaked the Lambda Director's Office
maptweak: tweaked the Lambda Administration Office
maptweak: tweaked the Lambda Relaxation Room
/:cl:

### CODE REVIEW

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -569,6 +569,10 @@
 	name = "crumpled note"
 	info = "<b>there is cotten candy in the walls</b>"
 
+/obj/item/paper/bigred/lambda
+	name = "ripped diary entry"
+	info = "Director Smith's Diary \n Entry Date: 15 December 20## \n Today.. I feel progress! The XX-121 species reproduction programm is in full effect and Administrator Cooper gave us the all clear to continue breeding them. It appears that this original XX-121 specimen is what we can call a 'Queen' of the hive... it's almost 5 metres tall and has already produced a bunch of eggs! This beauty will be the next Synthetic, we'll show those Seegson bastards."
```
Comment (Nanu308) [inline code/modules/paperwork/paper.dm:574]: Did you run this by the lore team? If not, reach out to them so they can give some input

**Review by Nanu308 — CHANGES_REQUESTED** [review]
see comments

File: maps/map_files/BigRed/sprinkles/25.vault_v2.dmm
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.vault_v2.dmm:1]: Also, use /noop/ for area and tiles where you are able.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/map_files/BigRed/sprinkles/25.vault_v2.dmm [hunk 2]
```diff
@@ -1,508 +1,609 @@
...
+	dir = 6;
+	icon_state = "whitegreen"
+	},
+/area/bigredv2/caves/lambda/xenobiology)
 "ag" = (
-/turf/open/mars{
-	icon_state = "mars_dirt_9"
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	health = 25000
```
[the remark this answers is not in the document]
Comment (BadAtThisGame302) [reply maps/map_files/BigRed/sprinkles/25.vault_v2.dmm:38]: I did not intend this, I didn't even mean to tune it I just used one of the Big Red ones that must also be editted aswell.

File: maps/map_files/BigRed/sprinkles/25.vault_v2.dmm
Comment (BadAtThisGame302) [reply maps/map_files/BigRed/sprinkles/25.vault_v2.dmm:1]: Will do.

File: code/modules/paperwork/paper.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply code/modules/paperwork/paper.dm]: Well I pinged the Lore Coordinator, I can't ping any lore contributors so I guess we will have to wait and see.

**Review by Nanu308 — APPROVED** [review]
lgtm

**Pull request merged**